p2p/gossip/protocol.go

```diff
@@ -2,405 +2,340 @@ package gossip
 
 import (
 	"errors"
-	"fmt"
-	"github.com/davecgh/go-spew/spew"
+	"github.com/golang/protobuf/proto"
 	"github.com/spacemeshos/go-spacemesh/crypto"
 	"github.com/spacemeshos/go-spacemesh/log"
 	"github.com/spacemeshos/go-spacemesh/p2p/config"
-	"github.com/spacemeshos/go-spacemesh/p2p/message"
-	"github.com/spacemeshos/go-spacemesh/p2p/net"
 	"github.com/spacemeshos/go-spacemesh/p2p/node"
+	"github.com/spacemeshos/go-spacemesh/p2p/pb"
+	"github.com/spacemeshos/go-spacemesh/p2p/service"
 	"hash/fnv"
 	"sync"
 	"time"
 )
 
-// NoResultsInterval is the timeout we wait between requesting more peers repeatedly
-const NoResultsInterval = 1 * time.Second
+const messageQBufferSize = 100
 
-// PeerMessageQueueSize is the len of the msgQ each peer holds.
-const PeerMessageQueueSize = 100
+const ProtocolName = "/p2p/1.0/gossip"
+const protocolVer = "0"
```

Comment:
you use this as ClientVersion in metadata. change to a valid var and possibly rename ?